### ISSUE TITLE
Cant create folders in subfolders of mqtt.*

### ISSUE THREAD

**Issue by mickym2** [ioBroker/ioBroker.mqtt#366]
I do not know when it has been changed - but at the moment it is only possible to create folders directly under mqtt.0 or mqtt.1 etc.

It worked in the past but at this moment no folder can be created down the hierarchy now.

**Comment by Apollon77** [comment]
Do you mean "manually via admin"? What exactly is not working? But then this is more an Admin issue?

**Comment by mickym2** [comment]
Yes via admin - it was working under admin 5.x within the mqtt instances. So this new restrictions are new - but I cannot say since when - as I check not each update of the admin if functionality is still available. As said long time ago - there should be no restrictions at all in the hierarchy unter mqtt.x as each state can have children and mqtt doesn't differentiate between folders. So in  early times I could create folders within folders. Now I can only create states within a folder. The only way to create subfolders is to create a folder with all hierarchies in mqtt.x- I.e. to create a subfolder I have to create within mqtt.0 a folder named folder.subfolder. Before it was possible to create within a folder another folder and states on each level.  I do not know when this has changed.

**Comment by Apollon77** [comment]
@GermanBluefox Was there any change?
